gradle/dependencies.gradle

```diff
@@ -15,12 +15,12 @@ ext {
       pollexor           : 'com.squareup:pollexor:2.0.4',
       supportV4          : "com.android.support:support-v4:$supportLibrariesVersion",
       supportAnnotations : "com.android.support:support-annotations:$supportLibrariesVersion",
-      junit              : 'junit:junit:4.10',
+      junit              : 'junit:junit:4.12',
       fest               : 'org.easytesting:fest-assert-core:2.0M10',
       festAndroid        : 'com.squareup:fest-android:1.0.6',
       robolectric        : 'org.robolectric:robolectric:3.1',
```

Comment:
what about `robolectric` and `okhtttp` libs upgrade?

Reply:
They break the unit tests. I didn't feel like making this too big a PR.